modules/gui/src/com/haulmont/cuba/gui/xml/layout/loaders/FormLoader.java

```diff
@@ -126,7 +126,20 @@ protected void loadColumns(Form resultComponent, Element element) {
             int colIndex = 0;
             for (Element columnElement : columnElements) {
                 String columnWidth = loadThemeString(columnElement.attributeValue("width"));
-                Iterable<Component> columnComponents = loadComponents(columnElement, columnWidth);
+
+                String flexAttr = columnElement.attributeValue("flex");
+                Float flex = null;
+                if (!Strings.isNullOrEmpty(flexAttr)) {
+                    try {
+                        flex = Float.parseFloat(flexAttr);
+                        if (flex > 0.0F) {
+                            resultComponent.setColumnFlex(colIndex, flex);
+                        }
+                    } catch (Exception e) {
```

Comment:
Please, remove `try-catch` and  `if (flex > 0.0F)`, because we don't catch parsing exceptions in loaders as well as don't filter entered values. If developers made a mistake, they must know about it)

e.g. see: https://github.com/cuba-platform/cuba/blob/master/modules/gui/src/com/haulmont/cuba/gui/xml/layout/loaders/GridLayoutLoader.java#L74